translations/de.ts

```diff
@@ -2493,12 +2498,12 @@ Leider führen deine Proxyeinstellungen zu Problemen. Bitte ändere deine Einste
     <message>
         <location filename="../src/widget/widget.cpp" line="1666"/>
         <source>Add new circle...</source>
-        <translation>Neuen Kreis hinzufügen...</translation>
+        <translation>Neuen Kreis hinzufügen ...</translation>
     </message>
     <message>
         <location filename="../src/widget/widget.cpp" line="1692"/>
         <source>By Name</source>
-        <translation>Nach Name</translation>
+        <translation>Nach Namen</translation>
```

Comment:
"Jede Anwendung, die ich kenne …"
"Jeder, den ich kenne klaut, also muss ich auch klauen."  ??
Überzeugt mich nicht. Nur weil andere es falsch machen, muss man es nicht selbst auc falsch machen.
Wenn du das korrekte "Namen" nicht magst, kann ich auch "Name" schreiben und das "nach" löschen. oder ich schreibe "Spalte" dazu, wie in deinem Beispiel.
Auf jeden Fall ist aber "Nach Name" falsch, und ich weigere mich in meinem PR absichtlich flasches einzubinden. "Nach Typ" ist korrekt.